Gulpfile.js

```diff
@@ -18,6 +18,7 @@ var del, livereload, runSequence;
 
 if (process.env.NODE_ENV == "development") {
   del         = require('del'),
+  colors      = require('colors'),
```

Comment:
Yeah, it wasn't required anywhere, but was being used on line [line 88](https://github.com/adorableio/avatars-api/blob/b2fb146568381bf9cb47367aa67aafdbc3f31246/Gulpfile.js#L188).